app/app.go

```diff
@@ -784,6 +793,14 @@ func NewGaiaApp(
 	return app
 }
 
+func GetDefaultBypassFeeMessages() []string {
+	return []string{
+		sdk.MsgTypeURL(&ibcchanneltypes.MsgRecvPacket{}),
```

Comment:
this makes think maybe in our `const.go` we should use `sdk.MsgTypeURL()` rather than directly a string.
for example:  use `sdk.MsgTypeURL(&ibcchanneltypes.MsgRecvPacket{})` rather than direclty `"/ibc.core.channel.v1.MsgRecvPacket"`